packages/@lightningjs/ui-components/src/components/Radio/Radio.js

```diff
@@ -67,18 +68,12 @@ export default class Radio extends Base {
       ? this.style.backgroundColorChecked
       : this.style.backgroundColor;
 
-    // if the inner body should be square, a rounded corner radius can still be applied to the stroke
-    const radius =
-      this.style.radius >= this.w / 2
-        ? (this.w - this.style.strokeWidth - 2) / 2
-        : 0;
-
     this._Body.patch({
       texture: lng.Tools.getRoundRect(
         // Compensating for the extra 2 pixels getRoundRect adds
         this.w - this.style.strokeWidth * 2 - 2,
         this.h - this.style.strokeWidth * 2 - 2,
-        radius,
+        getMaxRoundRadius(this.style.radius, this.w, this.h, 0),
```

Comment:
I think the width and height here need to match the value above `this.w - this.style.strokeWidth * 2` and `this.h - this.style.strokeWidth * 2`